test/core/extensions/coreIssuance.spec.ts

```diff
@@ -162,7 +162,7 @@ contract('CoreIssuance', accounts => {
 
     describe('when the set was not created through core', async () => {
       beforeEach(async () => {
-        subjectSetToIssue = NULL_ADDRESS;
+        subjectSetToIssue = SetProtocolUtils.CONSTANTS.NULL_ADDRESS;
```

Comment:
Maybe we can declare
```
const { NULL_ADDRESS }` =  SetProtocolUtils.CONSTANTS;
```
above, so we only need to reference NULL_ADDRESS in the tests themselves

Reply:
+1